boards/stk3700/dist/openocd.cfg

```diff
@@ -0,0 +1 @@
+source [find board/efm32.cfg]
```

Comment:
Nice. I think this change should work with all stk boards. I can try on slstk4201a.